src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8212,7 +8212,6 @@ private StatementSyntax ParseLocalDeclarationStatement()
             {
                 TypeSyntax type;
```

Comment:
This file contains only formatting changes. Consider reverting.

Reply:
Reverted

Reply:
It wasn't completely reverted (as of iteration 30).

---
In reply to: [203527181](https://github.com/dotnet/roslyn/pull/28587#discussion_r203527181) [](ancestors = 203527181)